homeassistant/components/shelly/cover.py

```diff
@@ -110,3 +139,61 @@ def _update_callback(self) -> None:
         """When device updates, clear control result that overrides state."""
         self.control_result = None
         super()._update_callback()
+
+
+class RpcShellyCover(ShellyRpcEntity, CoverEntity):
+    """Entity that controls a cover on RPC based Shelly devices."""
+
+    _attr_device_class = CoverDeviceClass.SHUTTER
+
+    def __init__(self, wrapper: RpcDeviceWrapper, id_: int) -> None:
+        """Initialize rpc cover."""
+        super().__init__(wrapper, f"cover:{id_}")
```

Comment:
Note that unique ids are per integration + platform so `cover:` shouldn't be needed here unless its needed for something else

Reply:
Currently it is not needed, but we already did it for the `switch` platform and history with Shelly they might make a model which need restore entity in the future than this key helps us to match the restored entity with the device